### DISCUSSION TITLE
Is something not working at Vercel right now?

### DISCUSSION THREAD

**Discussion by aorsten** [vercel/vercel#4997] [Help] [answered]
I tried to run `vercel --prod` to publish a NextJS project which already is in production. My only addition was `/public/sitemap.xml` with a simple sitemap file. I'm getting these build logs - which is very cryptic:

```
11:26:11.064 | Retrieving list of deployment files...
11:26:12.371 | Downloading 276 deployment files...
11:26:15.612 | Installing build runtime...
11:26:31.297 | Error: Exited with 1
11:26:31.297 | at ChildProcess.<anonymous> (/var/task/sandbox-worker.js:21953:21)
11:26:31.297 | at ChildProcess.emit (events.js:310:20)
11:26:31.297 | at ChildProcess.EventEmitter.emit (domain.js:482:12)
11:26:31.297 | at maybeClose (internal/child_process.js:1021:16)
11:26:31.297 | at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
11:26:33.645 | Done with "package.json"
```

In addition, when running `vercel --prod` I get the message: `UPDATE AVAILABLE Run npm i -g vercel@latest to install Vercel CLI 19.2.0`

However, running that npm command yields:

```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@vercel%2fbuild-utils - Not found
npm ERR! 404
npm ERR! 404  '@vercel/build-utils@2.4.2' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'vercel'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Any ideas what's going on?

**Comment by jpbow** [comment]
Deployments are broken for me too, and all serverless functions are returning 502 errors in production 🙈

**Comment by markozxuu** [comment, marked as answer]
Hi @jpbow  @aorsten ! These compilation problems are due to npm having trouble registering packages. Existing deployments and live traffic are not affected.

Please following to https://twitter.com/vercel_status

**Reply by styfle** [reply]
If you don't use twitter, you can subscribe to email updates at https://www.vercel-status.com
